store/tikv/tests/util_test.go

```diff
@@ -98,3 +100,17 @@ func (s *OneByOneSuite) TearDownSuite(c *C) {
 		withTiKVGlobalLock.RUnlock()
 	}
 }
+
+func encodeKey(prefix, s string) []byte {
```

Comment:
duplicate function make me unhappy to give LGTM

Reply:
I will remove one of them in the end.